fx-src/FirefoxStorage.js

```diff
@@ -72,7 +72,12 @@ const statements = {
   "listRecords": `
     SELECT record
       FROM collection_data
-        WHERE collection_name = :collection_name;`
+        WHERE collection_name = :collection_name;`,
+
+  "importData": `
+    REPLACE INTO collection_data (collection_name, record_id, record)
```

Comment:
> Therefore the code in the adapter is rather naive and brutal because its input comes prepared and filtered already.

Ah, gotcha. Could you clarify the docs/comments for the adapter to note that they don't need to do the timestamp filtering themselves?